### PR TITLE
Add blank and negated + small adjustments

### PR DESCRIPTION
**Pull request by nekevss**
Related to #11

Primarily implementing `blank` and `negated` functionality onto blank.

Renames: `duration_sign` -> `sign`

### CODE REVIEW

File: src/components/duration.rs [hunk 1]
```diff
@@ -857,23 +857,36 @@ impl Duration {
...
     #[must_use]
-    pub fn abs(&self) -> Self {
+    pub fn sign(&self) -> i32 {
+        duration_sign(&self.iter().collect())
+    }
+
+    /// Returns whether the current `Duration` is blank.
+    pub fn blank(&self) -> bool {
+        duration_sign(&self.iter().collect()) == 0
+    }
```
Comment (jedel1043) [inline src/components/duration.rs:870]: Would probably be better to name this `is_zero`, I think.

Comment (jedel1043) [reply src/components/duration.rs]: I was more talking about the function itself, since `duration_sign` returns zero only when `self` is zero.

Comment (nekevss) [reply src/components/duration.rs]: Oh, I'm blind lol yeah, that can work too.

**Review by jedel1043 — APPROVED** [review]
LGTM!

File: src/components/duration.rs (hunk 1 above)
Comment (nekevss) [reply src/components/duration.rs]: Glancing back through old temporal meeting minutes. It used to be called `isZero` but was opted off of, but I see no reason to not go with `isZero` as long as the equivalency is noted.

Comment (jedel1043) [reply src/components/duration.rs]: Yeah, `blank` is not very descriptive IMHO. We can add some comments when we rename spec methods just like in Boa.

**Pull request merged**